src/dum/meson.build

```diff
@@ -62,10 +62,12 @@ endif
 
 if not libsmumps.found()
   libgalahad_single_src += files('mumps.F90')
+  galahad_headers += files('include/smumps_struc.h')
```

Comment:
Dummy header that we will install only if we didn't find the single precision version of `MUMPS`.